plugins/window/guest-js/index.ts

```diff
@@ -900,6 +900,30 @@ class WindowManager extends WebviewWindowHandle {
     });
   }
 
+  /**
+   * Set window effects.
+   *
+   * @since 2.0
+   */
+  async setEffects(effects: Effects): Promise<void> {
+    return window.__TAURI_INVOKE__("plugin:window|set_effects", {
+      label: this.label,
+      value: effects,
+    });
+  }
+
+  /**
+   * Clear any applied effects if possible.
+   *
+   * @since 2.0
+   */
+  async clearEffects(): Promise<void> {
```

Comment:
Should we add an explicit clear effects method on tauri's side? Instead of relying on None?

Reply:
I think setting None is quite idiomatic for Rust isn't it? For JavaScript it looks weird..